app/static/js/script.js

```diff
@@ -206,7 +206,7 @@ const mobileMod = `
 		<ul class="works-badges mobile modal">
 			${renderLanguagesLi(portfolioCard.badge1)}
 		</ul>
-		<img src="${modalWindow.mobileImage}" alt="background image">
+		<img src="${modalWindow.mobileImage}" alt="background image" id="mobile-modal-img">
```

Comment:
- [ ] Kindly align the `languages` to the left of the modal to start in the same line with the paragraph to the left as it is in the Figma design

Your design
![image](https://user-images.githubusercontent.com/81932478/190640421-1c591e50-a534-4f64-a81c-59201c9b4560.png)

Figma design
![image](https://user-images.githubusercontent.com/81932478/190640488-3a0050b1-dd04-4466-9665-c281aba08027.png)




- [ ] Secondly, it would be nice if you could add some `margin` and `border-radius` to the bottom of the `see project` buttons so as to create some space and have the mobile view of the modal match that of the given design

Your design
![image](https://user-images.githubusercontent.com/81932478/190641132-d7dcc88a-8355-4a63-80fa-5ba6e820cf58.png)

Figma design
![image](https://user-images.githubusercontent.com/81932478/190641237-ff7dd8fc-d828-47fc-ab66-f9ef2ad8f8d8.png)